scirius/settings.py

```diff
@@ -240,7 +240,7 @@
 # For basic authentication you can use
 # PROXY_PARAMS = { 'http': "http://user:pass@proxy:3128", 'https': "http://user:pass@proxy:3128" }
 
-GIT_SOURCES_BASE_DIRECTORY = os.path.join(BASE_DIR, 'git-sources/')
+GIT_SOURCES_BASE_DIRECTORY = os.path.join(BASE_DIR, 'data', 'git-sources/')
```

Comment:
humm, this will likely break compatibility for the people that have already running instances but did not change the path. I like the idea so maybe we could have, detection of old path and if not existing then we use the data version that you are proposing ?